### ISSUE TITLE
how  about the result?

### ISSUE THREAD

**Issue by wuqiangch** [johnkorn/speaker_recognition#1]
how about the result?

**Comment by johnkorn** [comment]
well, I got 97%/93% accuracy on train/validation sets, but I did not launch training on full LibriSpeech dataset, only on clean audios, so the model is not ready for real life usage yet. Thanks for the interest though.

**Comment by ovninosa** [comment]
@johnkorn there is a tutorial to run your proyect?

**Comment by wuqiangch** [comment]
@johnkorn Have you read the paper "Deep Speaker: an End-to-End Neural Speaker Embedding System"?Maybe the method is the best one of speaker recognition.

**Comment by johnkorn** [comment]
@javenosa no, I did not finish the work on the project because had to switch to other tasks, so no tutorial yet. 

@wuqiangch  thanks for the paper, looks like a good approach, exactly what I needed. I'll try to use it when come back to the project.
